cookiecutter.json

```diff
@@ -6,13 +6,15 @@
     "author_email": "",
     "license": ["BSD 3-Clause", "GNU GPL v3+", "Apache Software License 2.0", "BSD 2-Clause", "Other"],
     "project_url": "",
-    "minimum_python_version": ["3.7", "3.6", "3.5"],
+    "github_repo": "sunpy/{{ cookiecutter.package_name }}",
+    "minimum_python_version": ["3.8", "3.7", "3.6", "3.5"],
```

Comment:
Is 3.5 worth it?

Reply:
I have kept it because people might have old packages they haven't updated yet